docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,657 @@
+     "output_type": "stream",
+     "text": [
+      "The basket comprises Algeria's Saharan Blend, Indonesia's Minas, Nigeria's Bonny Light, Saudi Arabia's Arabian Light, \u001b[31mDubai\u001b[0m of the Uae, Venezuela's Tia Juana and Mexico's Isthmus.\n"
+     ]
+    }
+   ],
+   "source": [
+    "print_issue(issues[8], show_labels=False) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d213b2b2",
+   "metadata": {},
+   "source": [
+    "## 4. Most common word-level token mislabels \n",
+    "\n",
+    "It may be useful to examine the most common word-level token mislabels. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "f6bbed94",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "'Division' is mislabeled 30 times\n",
+      "-----------------------------\n",
+      "labeled as O, but predicted as MISC 30 times\n",
+      "\n",
+      "'Czech' is mislabeled 13 times\n",
+      "-----------------------------\n",
+      "labeled as LOC, but predicted as MISC 12 times\n",
+      "labeled as ORG, but predicted as LOC 1 times\n",
+      "\n",
+      "'League' is mislabeled 12 times\n",
+      "-----------------------------\n",
+      "labeled as O, but predicted as MISC 8 times\n",
+      "labeled as ORG, but predicted as MISC 2 times\n",
+      "labeled as LOC, but predicted as O 1 times\n",
+      "labeled as LOC, but predicted as ORG 1 times\n",
+      "\n",
+      "'Conference' is mislabeled 10 times\n",
+      "-----------------------------\n",
+      "labeled as O, but predicted as MISC 10 times\n",
+      "\n",
+      "'Hockey' is mislabeled 10 times\n",
+      "-----------------------------\n",
+      "labeled as O, but predicted as MISC 5 times\n",
+      "labeled as ORG, but predicted as MISC 5 times\n",
+      "\n",
+      "'National' is mislabeled 9 times\n",
+      "-----------------------------\n",
+      "labeled as ORG, but predicted as MISC 8 times\n",
+      "labeled as O, but predicted as MISC 1 times\n",
+      "\n",
+      "'Alpine' is mislabeled 8 times\n",
+      "-----------------------------\n",
+      "labeled as O, but predicted as MISC 8 times\n",
+      "\n",
+      "'Western' is mislabeled 8 times\n",
+      "-----------------------------\n",
+      "labeled as O, but predicted as MISC 7 times\n",
+      "labeled as O, but predicted as LOC 1 times\n",
+      "\n",
+      "'G' is mislabeled 6 times\n",
+      "-----------------------------\n",
+      "labeled as O, but predicted as MISC 6 times\n",
+      "\n",
+      "'Union' is mislabeled 6 times\n",
+      "-----------------------------\n",
+      "labeled as ORG, but predicted as MISC 6 times\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "words = [word for words in given_words for word in words]  \n",
+    "frequency = frequent_words(issues, words, labels, pred_probs) \n",
+    "show_frequent_issues(frequency, entities, verbose=True) "
```

Comment:
Would it be possible to merge these two into a single function, or do you think that's too messy?  
It's a bit annoying for user to have to call two functions.

Reply:
Let's also rename the function: `show_frequent_issues` -> `common_token_issues`